.github/workflows/release-packages.yaml

```diff
@@ -118,6 +118,7 @@ jobs:
 
 
   homebrew-pr:
+    if: ${{ false }}  # disable for now
```

Comment:
Can we please immediately have a PR that will re-enable this (once the PR is merged)? I am _very_ worried about us forgetting about this.

Reply:
I can do that immediately then if that eases your mind (even though it's on my personal todo list as a step after getting it working, so I wouldn't forget about it).

I'll submit a new PR reversing this and link to it here.

Reply:
Okay the new PR is available at https://github.com/diffblue/cbmc/pull/5748.

Please do not merge it yet - I'll merge it when the dockerhub image push is ready.

Reply:
Thank you @NlightNFotis for creating the follow-up PR!